includes/RequireGutenberg.php

```diff
@@ -68,7 +68,7 @@ public function admin_notice() {
 			return;
 		}
 		?>
-		<div class="notice notice-warning require-gutenberg-notice-wrapper notice-alt<?php echo ( $active_step ) ? ' active-step-' . esc_attr( $active_step ) : ''; ?>">
+		<div class="notice notice-warning require-gutenberg-notice-wrapper notice-alt<?php echo ' active-step-' . esc_attr( $active_step ); ?>">
```

Comment:
Why this change? :thinking:

Reply:
`$active_step` is checked above thus it is not possible to be falsy

Reply:
Good point, I missed it. Thanks!

Reply:
@phpstan does not miss what we humans overlook.